src/browser/AppInit.jsx

```diff
@@ -90,14 +90,14 @@ const AppInit = () => {
   return (
     <Provider store={store}>
       <BusProvider bus={bus}>
-        <React.Fragment>
+        <>
```

Comment:
I know these are interchangeable, so do we generally prefer `<>` to `<React.Fragment`?

Reply:
Yeah, the linter will fix it automatically. Do you mind that rule?

Reply:
Im fine with this, I wonder what anyone else feels in terms of readability? Otherwise no problem from me

Reply:
It's quite common so I don't think it will be a problem